db/db.go

```diff
@@ -80,62 +69,45 @@ func Open(dbPath string) (*DB, error) {
 	openMu.Lock()
 	defer openMu.Unlock()
 
-	dbc, err := sql.Open("sqlite3_with_hook", dbPath)
+	os.MkdirAll(filepath.Dir(dbPath), 0755)
```

Comment:
Apparently sql.Open creates the directory. I just tested again, and this is the error if we don't create the directory:
```2016/03/14 21:05:39 failed to open store: unable to open database file```